lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -107,196 +107,201 @@ <h3 class="course-title" id="course-title-${enrollment.course_id}">
             <span>${course_overview.display_name_with_default}</span>
           % endif
         </h3>
-        <div class="course-info">
-          <span class="info-university">${course_overview.display_org_with_default} - </span>
-          <span class="info-course-id">${course_overview.display_number_with_default}</span>
-          <%
-            enrollment_date = course_overview.self_paced and enrollment and enrollment.created
-            if course_overview.start_date_is_still_default:
-                container_string = _("Coming Soon")
-                course_date = None
-            else:
-                format = 'shortDate'
-                dashboard_start_display = course_overview.dashboard_start_display
-                if course_overview.has_ended():
-                    container_string = _("Ended - {date}")
-                    course_date = course_overview.end
-                elif course_overview.has_started():
-                    container_string = _("Started - {date}")
-                    if enrollment_date and isinstance(dashboard_start_display, datetime.datetime):
-                      course_date = max(enrollment_date, dashboard_start_display)
-                    else:
-                      course_date = enrollment_date or dashboard_start_display
-                elif course_overview.starts_within(days=5):
-                    container_string = _("Starts - {date}")
-                    course_date = dashboard_start_display
-                    format = 'defaultFormat'
-                else: ## hasn't started yet
-                    container_string = _("Starts - {date}")
-                    course_date = dashboard_start_display
+        <div class="wrapper-course-info-actions">
+          <div class="wrapper-course-info-row">
```

Comment:
wrapper-course-info-actions and wrapper-course-info-row are net new. 
<img width="487" alt="Screen Shot 2021-03-30 at 10 20 28 AM" src="https://user-images.githubusercontent.com/34042537/113004668-c7727180-9141-11eb-90ce-0ea32ef84bfb.png">